db/migrate/20230722114414_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :body
```

Comment:
- [ ] Please call this column `text` instead of `body`

Reply:
- [ ] Remember to add `comments_counter` and `likes_counter` to your table

Reply:
- [ ] Please create a migration file for creating a comments table.
- [ ] Create likes table